### PR TITLE
Add OdsOrganisations schema and tables

### PR DESCRIPTION
**Pull request by bjss-colin**
image

### CODE REVIEW

File: database/NHSD.GPITBuyingCatalogue.Database/OdsOrganisations/Tables/OdsOrganisations.sql [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    [AddressLine1]          NVARCHAR(255) NULL,
+    [AddressLine2]          NVARCHAR(255) NULL,
+    [AddressLine3]          NVARCHAR(255) NULL,
+    [AddressLine4]          NVARCHAR(255) NULL,
+    [AddressLine5]          NVARCHAR(255) NULL,
+    [Town]                  NVARCHAR(100) NULL,
+    [County]                NVARCHAR(100) NULL,
+    [Postcode]              NVARCHAR(10)  NULL,
+    [Country]               NVARCHAR(100) NULL,
+    [Status]                NVARCHAR(100) NOT NULL,
```
Comment (magnusjohn-bjss) [inline database/NHSD.GPITBuyingCatalogue.Database/OdsOrganisations/Tables/OdsOrganisations.sql]: Doesn't the status field only hold active/inactive values? Would this be better as an IsActive flag?

File: database/NHSD.GPITBuyingCatalogue.Database/OdsOrganisations/Tables/OrganisationRelationships.sql [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿CREATE TABLE [ods_organisations].[OrganisationRelationships]
+(
+    [UniqueRelId]               INT             NOT NULL PRIMARY KEY,
```
Comment (magnusjohn-bjss) [inline database/NHSD.GPITBuyingCatalogue.Database/OdsOrganisations/Tables/OrganisationRelationships.sql]: Should this be an identity field? Also "Id" is probly fine as a field name, don't think we qualify entity ids anywhere else

File: database/NHSD.GPITBuyingCatalogue.Database/OdsOrganisations/Tables/OrganisationRoles.sql [hunk 3]
```diff
@@ -0,0 +1,9 @@
+﻿CREATE TABLE [ods_organisations].[OrganisationRoles]
+(
+    [UniqueRoleId]      INT             NOT NULL PRIMARY KEY,
```
Comment (magnusjohn-bjss) [inline database/NHSD.GPITBuyingCatalogue.Database/OdsOrganisations/Tables/OrganisationRoles.sql]: Same as above

File: database/NHSD.GPITBuyingCatalogue.Database/OdsOrganisations/Tables/OdsOrganisations.sql (hunk 1 above)
Comment (ghost) [reply database/NHSD.GPITBuyingCatalogue.Database/OdsOrganisations/Tables/OdsOrganisations.sql]: Column renamed and datatype changed to BIT

File: database/NHSD.GPITBuyingCatalogue.Database/OdsOrganisations/Tables/OrganisationRelationships.sql (hunk 2 above)
Comment (ghost) [reply database/NHSD.GPITBuyingCatalogue.Database/OdsOrganisations/Tables/OrganisationRelationships.sql]: As mentioned on Slack, these will be mapped directly via XML attributes. They will be unique but aren't generated by SQL or in code.

File: database/NHSD.GPITBuyingCatalogue.Database/OdsOrganisations/Tables/OrganisationRoles.sql (hunk 3 above)
Comment (ghost) [reply database/NHSD.GPITBuyingCatalogue.Database/OdsOrganisations/Tables/OrganisationRoles.sql]: Same response as above

**Review by magnusjohn-bjss — DISMISSED** [review]

**Review by magnusjohn-bjss — APPROVED** [review]

**Pull request merged**